var/spack/repos/builtin/packages/py-h5py/package.py

```diff
@@ -15,6 +15,7 @@ class PyH5py(PythonPackage):
 
     import_modules = ['h5py', 'h5py._hl']
 
+    version('2.9.0', sha256='9d41ca62daf36d6b6515ab8765e4c8c4388ee18e2a665701fef2b41563821002')
```

Comment:
Any particular reason for this?

Reply:
Yes, I need support for float128. Tests fail with 2.8.0

Reply:
OK